src/stories/Library/disclosure/Disclosure.tsx

```diff
@@ -45,13 +45,13 @@ export const Disclosure: React.FC<DisclosureProps> = ({
             />
           </div>
         )}
-        <span
-          className={`disclosure__text ${
+        <h2
```

Comment:
The disclosure story is used multiple places in the design system and it seems like a stretch to assume that h2 is the right heading level everywhere in the deisgn

I think we need some way to control this from the outside. Here is a seemingly simple way to achieve this using a prop: https://dev.to/s_aitchison/psa-stop-hard-coding-heading-levels-in-your-react-components-2ekp. Note that it probably has to be adjusted to TypeScript and our use case.